calculatorUICOOP/ViewModels/MainPageViewModel.cs

```diff
@@ -27,31 +27,49 @@ public string DisplayContent
             }
         }
         #endregion
-        
+
         #region Commands
-        public ICommand numberInputCommand { get; set; }
-        // todo: create a command for operators. Do we need a command parameter?
-        // todo: create a command for "clear". Again... do we need a command parameter?
+        public ICommand NumberInputCommand { get; set; }
+        public ICommand ClearInputCommand { get; set; }
+        public ICommand DeleteInputCommand { get; set; }
+        public ICommand DivideInputCommand { get; set; }
+        public ICommand MultiplyInputCommand { get; set; }
```

Comment:
I would make all 4 (add, subtract, multiply, divide) into a single command with a command parameter that has a string. Then use a switch statement to determine the operator from the command parameter and do that operation.